[2024-12-12 21:28:02,067] ERROR log_exception() L875  Exception on /escala [GET] - call_trace=C:\Users\maria\OneDrive\Documentos\Cursos\Testes\desenvolvimento\basico\aula-3-novo\meu_app_api\env\Lib\site-packages\flask\app.py L875 
Traceback (most recent call last):
  File "C:\Users\maria\OneDrive\Documentos\Cursos\Testes\desenvolvimento\basico\aula-3-novo\meu_app_api\env\Lib\site-packages\flask\app.py", line 1511, in wsgi_app
    response = self.full_dispatch_request()
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\maria\OneDrive\Documentos\Cursos\Testes\desenvolvimento\basico\aula-3-novo\meu_app_api\env\Lib\site-packages\flask\app.py", line 919, in full_dispatch_request
    rv = self.handle_user_exception(e)
         ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\maria\OneDrive\Documentos\Cursos\Testes\desenvolvimento\basico\aula-3-novo\meu_app_api\env\Lib\site-packages\flask_cors\extension.py", line 194, in wrapped_function
    return cors_after_request(app.make_response(f(*args, **kwargs)))
                                                ^^^^^^^^^^^^^^^^^^
  File "C:\Users\maria\OneDrive\Documentos\Cursos\Testes\desenvolvimento\basico\aula-3-novo\meu_app_api\env\Lib\site-packages\flask\app.py", line 917, in full_dispatch_request
    rv = self.dispatch_request()
         ^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\maria\OneDrive\Documentos\Cursos\Testes\desenvolvimento\basico\aula-3-novo\meu_app_api\env\Lib\site-packages\flask\app.py", line 902, in dispatch_request
    return self.ensure_sync(self.view_functions[rule.endpoint])(**view_args)  # type: ignore[no-any-return]
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\maria\OneDrive\Documentos\Cursos\Testes\desenvolvimento\basico\aula-3-novo\meu_app_api\env\Lib\site-packages\flask_openapi3\scaffold.py", line 117, in view_func
    response = func(**func_kwargs)
               ^^^^^^^^^^^^^^^^^^^
  File "C:\Users\maria\OneDrive\Documentos\Cursos\Testes\desenvolvimento\basico\aula-3-novo\meu_app_api\app.py", line 94, in get_escala
    escala_id = query.id
                ^^^^^^^^
  File "C:\Users\maria\OneDrive\Documentos\Cursos\Testes\desenvolvimento\basico\aula-3-novo\meu_app_api\env\Lib\site-packages\pydantic\main.py", line 892, in __getattr__
    raise AttributeError(f'{type(self).__name__!r} object has no attribute {item!r}')
AttributeError: 'EscalaBuscaSchema' object has no attribute 'id'
[2024-12-12 21:28:02,163] INFO _log() L97   127.0.0.1 - - [12/Dec/2024 21:28:02] "[35m[1mGET /escala?nome_funcionario=Mariana HTTP/1.1[0m" 500 - - call_trace=C:\Users\maria\OneDrive\Documentos\Cursos\Testes\desenvolvimento\basico\aula-3-novo\meu_app_api\env\Lib\site-packages\werkzeug\_internal.py L97  
[2024-12-12 21:34:17,579] INFO _log() L97   [31m[1mWARNING: This is a development server. Do not use it in a production deployment. Use a production WSGI server instead.[0m
 * Running on all addresses (0.0.0.0)
 * Running on http://127.0.0.1:5000
 * Running on http://192.168.15.4:5000 - call_trace=C:\Users\maria\OneDrive\Documentos\Cursos\Testes\desenvolvimento\basico\aula-3-novo\meu_app_api\env\Lib\site-packages\werkzeug\_internal.py L97  
[2024-12-12 21:34:17,582] INFO _log() L97   [33mPress CTRL+C to quit[0m - call_trace=C:\Users\maria\OneDrive\Documentos\Cursos\Testes\desenvolvimento\basico\aula-3-novo\meu_app_api\env\Lib\site-packages\werkzeug\_internal.py L97  
[2024-12-12 21:36:31,719] INFO _log() L97   127.0.0.1 - - [12/Dec/2024 21:36:31] "GET /openapi/swagger HTTP/1.1" 200 - - call_trace=C:\Users\maria\OneDrive\Documentos\Cursos\Testes\desenvolvimento\basico\aula-3-novo\meu_app_api\env\Lib\site-packages\werkzeug\_internal.py L97  
[2024-12-12 21:36:31,878] INFO _log() L97   127.0.0.1 - - [12/Dec/2024 21:36:31] "[36mGET /openapi/swagger/css/swagger-ui.css HTTP/1.1[0m" 304 - - call_trace=C:\Users\maria\OneDrive\Documentos\Cursos\Testes\desenvolvimento\basico\aula-3-novo\meu_app_api\env\Lib\site-packages\werkzeug\_internal.py L97  
[2024-12-12 21:36:31,881] INFO _log() L97   127.0.0.1 - - [12/Dec/2024 21:36:31] "[36mGET /openapi/swagger/js/swagger-ui-bundle.js HTTP/1.1[0m" 304 - - call_trace=C:\Users\maria\OneDrive\Documentos\Cursos\Testes\desenvolvimento\basico\aula-3-novo\meu_app_api\env\Lib\site-packages\werkzeug\_internal.py L97  
[2024-12-12 21:36:31,887] INFO _log() L97   127.0.0.1 - - [12/Dec/2024 21:36:31] "[36mGET /openapi/swagger/js/swagger-ui-standalone-preset.js HTTP/1.1[0m" 304 - - call_trace=C:\Users\maria\OneDrive\Documentos\Cursos\Testes\desenvolvimento\basico\aula-3-novo\meu_app_api\env\Lib\site-packages\werkzeug\_internal.py L97  
[2024-12-12 21:36:32,150] INFO _log() L97   127.0.0.1 - - [12/Dec/2024 21:36:32] "GET /openapi/openapi.json HTTP/1.1" 200 - - call_trace=C:\Users\maria\OneDrive\Documentos\Cursos\Testes\desenvolvimento\basico\aula-3-novo\meu_app_api\env\Lib\site-packages\werkzeug\_internal.py L97  
[2024-12-12 21:36:32,152] INFO _log() L97   127.0.0.1 - - [12/Dec/2024 21:36:32] "[36mGET /openapi/swagger/images/swagger.svg HTTP/1.1[0m" 304 - - call_trace=C:\Users\maria\OneDrive\Documentos\Cursos\Testes\desenvolvimento\basico\aula-3-novo\meu_app_api\env\Lib\site-packages\werkzeug\_internal.py L97  
[2024-12-12 21:36:44,967] ERROR log_exception() L875  Exception on /escala [GET] - call_trace=C:\Users\maria\OneDrive\Documentos\Cursos\Testes\desenvolvimento\basico\aula-3-novo\meu_app_api\env\Lib\site-packages\flask\app.py L875 
Traceback (most recent call last):
  File "C:\Users\maria\OneDrive\Documentos\Cursos\Testes\desenvolvimento\basico\aula-3-novo\meu_app_api\env\Lib\site-packages\flask\app.py", line 1511, in wsgi_app
    response = self.full_dispatch_request()
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\maria\OneDrive\Documentos\Cursos\Testes\desenvolvimento\basico\aula-3-novo\meu_app_api\env\Lib\site-packages\flask\app.py", line 919, in full_dispatch_request
    rv = self.handle_user_exception(e)
         ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\maria\OneDrive\Documentos\Cursos\Testes\desenvolvimento\basico\aula-3-novo\meu_app_api\env\Lib\site-packages\flask_cors\extension.py", line 194, in wrapped_function
    return cors_after_request(app.make_response(f(*args, **kwargs)))
                                                ^^^^^^^^^^^^^^^^^^
  File "C:\Users\maria\OneDrive\Documentos\Cursos\Testes\desenvolvimento\basico\aula-3-novo\meu_app_api\env\Lib\site-packages\flask\app.py", line 917, in full_dispatch_request
    rv = self.dispatch_request()
         ^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\maria\OneDrive\Documentos\Cursos\Testes\desenvolvimento\basico\aula-3-novo\meu_app_api\env\Lib\site-packages\flask\app.py", line 902, in dispatch_request
    return self.ensure_sync(self.view_functions[rule.endpoint])(**view_args)  # type: ignore[no-any-return]
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\maria\OneDrive\Documentos\Cursos\Testes\desenvolvimento\basico\aula-3-novo\meu_app_api\env\Lib\site-packages\flask_openapi3\scaffold.py", line 117, in view_func
    response = func(**func_kwargs)
               ^^^^^^^^^^^^^^^^^^^
  File "C:\Users\maria\OneDrive\Documentos\Cursos\Testes\desenvolvimento\basico\aula-3-novo\meu_app_api\app.py", line 94, in get_escala
    escala_id = query.id
                ^^^^^^^^
  File "C:\Users\maria\OneDrive\Documentos\Cursos\Testes\desenvolvimento\basico\aula-3-novo\meu_app_api\env\Lib\site-packages\pydantic\main.py", line 892, in __getattr__
    raise AttributeError(f'{type(self).__name__!r} object has no attribute {item!r}')
AttributeError: 'EscalaBuscaSchema' object has no attribute 'id'
[2024-12-12 21:36:44,984] INFO _log() L97   127.0.0.1 - - [12/Dec/2024 21:36:44] "[35m[1mGET /escala?nome_funcionario=Mariana HTTP/1.1[0m" 500 - - call_trace=C:\Users\maria\OneDrive\Documentos\Cursos\Testes\desenvolvimento\basico\aula-3-novo\meu_app_api\env\Lib\site-packages\werkzeug\_internal.py L97  
[2024-12-12 21:40:58,648] INFO _log() L97   [31m[1mWARNING: This is a development server. Do not use it in a production deployment. Use a production WSGI server instead.[0m
 * Running on all addresses (0.0.0.0)
 * Running on http://127.0.0.1:5000
 * Running on http://192.168.15.4:5000 - call_trace=C:\Users\maria\OneDrive\Documentos\Cursos\Testes\desenvolvimento\basico\aula-3-novo\meu_app_api\env\Lib\site-packages\werkzeug\_internal.py L97  
[2024-12-12 21:40:58,651] INFO _log() L97   [33mPress CTRL+C to quit[0m - call_trace=C:\Users\maria\OneDrive\Documentos\Cursos\Testes\desenvolvimento\basico\aula-3-novo\meu_app_api\env\Lib\site-packages\werkzeug\_internal.py L97  
[2024-12-12 21:41:07,952] INFO _log() L97   127.0.0.1 - - [12/Dec/2024 21:41:07] "[32mGET / HTTP/1.1[0m" 302 - - call_trace=C:\Users\maria\OneDrive\Documentos\Cursos\Testes\desenvolvimento\basico\aula-3-novo\meu_app_api\env\Lib\site-packages\werkzeug\_internal.py L97  
[2024-12-12 21:41:07,981] INFO _log() L97   127.0.0.1 - - [12/Dec/2024 21:41:07] "GET /openapi/ HTTP/1.1" 200 - - call_trace=C:\Users\maria\OneDrive\Documentos\Cursos\Testes\desenvolvimento\basico\aula-3-novo\meu_app_api\env\Lib\site-packages\werkzeug\_internal.py L97  
[2024-12-12 21:41:08,300] INFO _log() L97   127.0.0.1 - - [12/Dec/2024 21:41:08] "[36mGET /openapi/rapipdf/images/rapipdf.svg HTTP/1.1[0m" 304 - - call_trace=C:\Users\maria\OneDrive\Documentos\Cursos\Testes\desenvolvimento\basico\aula-3-novo\meu_app_api\env\Lib\site-packages\werkzeug\_internal.py L97  
[2024-12-12 21:41:08,309] INFO _log() L97   127.0.0.1 - - [12/Dec/2024 21:41:08] "[36mGET /openapi/redoc/images/redoc.svg HTTP/1.1[0m" 304 - - call_trace=C:\Users\maria\OneDrive\Documentos\Cursos\Testes\desenvolvimento\basico\aula-3-novo\meu_app_api\env\Lib\site-packages\werkzeug\_internal.py L97  
